src/get-started/codelab.md

```diff
@@ -78,9 +80,10 @@ The animated GIF shows how the app works at the completion of part 1.
 
   * A physical device ([Android][] or [iOS][]) connected to your
     computer and set to developer mode
+  * Screen reader enabled on the physical device (<a href="#talkback">TalkBack</a> on Android, <a href="#talkback">VoiceOver</a> on iPhone)
   * The [iOS simulator][] (requires installing Xcode tools)
   * The [Android emulator][] (requires setup in Android Studio)
-  * A browser (Chrome is required for debugging)
+  * A browser (Debugging requires the Chrome browser)
   * A [Windows][], [Linux][], or [macOS][] desktop application
```

Comment:
@Sfshaza and @khanhnwin, can you make this change? it requires updating the entire list, which I did not modify or test in my work.

Reply:
Thanks for looping me in! Let's shelve this for now. Since it's independent of the accessibility content, we should review with @sfshaza2 separately and open a different PR to update the list structure.